README.md

```diff
@@ -1,67 +1,66 @@
-﻿# Project Reunion
+﻿# Project Reunion - Calling all Windows desktop apps!
 
-[![Feature Proposals](https://img.shields.io/github/issues/microsoft/projectreunion/feature%20proposal)](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+label%3A%22feature+proposal%22)
-[![Bugs](https://img.shields.io/github/issues/microsoft/projectreunion/bug)](https://github.com/microsoft/ProjectReunion/issues?q=is%3Aissue+is%3Aopen+label%3Abug)
+| Delight users with modern UI | Access new platform features | Down-level support | 
+|:--:|:--:|:--:|
+| <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-get-started-get-started.svg" width=250 alt="Modern navigation"/></br>Powerful WinUI 3 UI | <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-get-started-scenarios.svg" width=250 alt="Platform logo"/></br>Localization + power status + more</br> | <img src="https://docs.microsoft.com/en-us/media/illustrations/biztalk-host-integration-install-configure.svg" width=250 alt="Down-level logo"/></br>Back to Windows 10 1809 |
 
+...and more, **without having to rewrite your app!**
 
-### What is Project Reunion?
-Project Reunion is our vision for unifying and evolving the Windows developer platform to make it easier
-to build great apps that work across all the Windows 10 versions and devices people use.
+* Use your current installer *(no requirement to use MSIX, but there are [reliability/security benefits to using MSIX](https://docs.microsoft.com/windows/msix/overview#key-features)!)*

```

Comment:
Maybe change to:

* Use your current installer - MSIX, setup.exe, MSI, or nearly anything else

Reply:
(There's clearly benefits to MSIX to be had, but maybe that goes into some future "Building Great Apps on Windows" content.)

Reply:
I believe we need to explicitly call out that there's no need/requirement for MSIX since traditionally we've always had this caveat, we've traditionally been like "WPF apps can use our new features!" *(if they're using MISX)*. We need to establish the trust, that we're serious, we really mean no MSIX needed.

Reply:
Fair. Maybe `(MSIX not required)`?